packages/manager/apps/hpc-vmware-managed-vcd/public/translations/datacentres/vrack-segment/Messages_fr_FR.json

```
@@ -5,6 +5,8 @@
   "managed_vcd_dashboard_vrack_description": "Les segments vRack facilitent l'organisation et l'interconnexion de vos différents services OVHcloud avec votre vDC. Une fois votre vDC associé à un vRack, vous disposez par défaut de 5 segments pour structurer votre infrastructure et segmenter efficacement votre réseau entre les produits OVHcloud.",
   "managed_vcd_dashboard_vrack_edit_vlan": "Modifier l'ID VLAN",
   "managed_vcd_dashboard_vrack_add_network": "Ajouter un sous-réseau",
+  "managed_vcd_dashboard_vrack_sub_network_address": "Adresse du sous-réseau",
+  "managed_vcd_dashboard_vrack_sub_network": "Sous-réseau",
```

Not sure sub_network is needed, but if it is, we should include it wherever network is used
```suggestion
  "managed_vcd_dashboard_vrack_network_address": "Adresse du sous-réseau",
  "managed_vcd_dashboard_vrack_network": "Sous-réseau",
```